frontend/src/components/AppRoutes.jsx

```diff
@@ -3,6 +3,7 @@ import React from "react"
 import { Route, Routes } from "react-router-dom"
 import NewBookForm from "../features/books/NewBookForm"
 import BookDetails from "../features/books/BookDetails"
+import EditBookForm from "../features/books/EditBookForm"
 
 
```

Comment:
@nshelokar-vf, why these unwanted lines?

Reply:
Removed it.